src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/exception.css.twig

```diff
@@ -1,99 +1,31 @@
-.sf-reset .traces {
-    padding: 0 0 1em 1.5em;
-}
-.sf-reset .traces a {
-    font-size: 14px;
-}
-.sf-reset .traces abbr {
-    border-bottom-color: #AAA;
-    padding-bottom: 2px;
-}
-.sf-reset .traces li {
-    color: #222;
-    font-size: 14px;
-    padding: 5px 0;
-    list-style-type: decimal;
-    margin: 0 0 0 1em;
-}
-.sf-reset .traces li.selected {
-    background: rgba(255, 255, 153, 0.5);
-}
+{{ include('@Twig/exception.css.twig') }}
```

Comment:
According to the [source](https://twig.sensiolabs.org/doc/1.x/functions/source.html) function documentation:
> The source function returns the content of a template without rendering it

> The function uses the same template loaders as the ones used to include templates. So, if you are using the filesystem loader, the templates are looked for in the paths defined by it.

Isn't this what we need?